composer/loggers/remote_uploader_downloader.py

```diff
@@ -163,16 +187,24 @@ class RemoteUploaderDownloader(LoggerDestination):
     """
 
     def __init__(self,
-                 object_store_cls: Type[ObjectStore],
-                 object_store_kwargs: Dict[str, Any],
-                 object_name: str = '{remote_file_name}',
+                 remote_bucket_uri: str,
+                 remote_backend_kwargs: Optional[Dict[str, Any]] = None,
+                 remote_path_format_string: str = '{remote_file_name}',
```

Comment:
given the class already says `RemoteUploaderDownloader`, can we remove the `remote_` prefix from these variables? They're a bit long and I don't think the prefix adds any necessary clarity since its already pretty clearly scoped

Reply:
hm ok. I think I always have a preference for long and extra clear, but I don't feel strongly about this one.